test/check-application

```diff
@@ -23,6 +23,9 @@ class TestApplication(testlib.MachineCase):
         # When we disconnect while certmonger wants to tell us something
         self.allow_journal_messages("org.fedorahosted.certmonger: couldn't introspect /org/fedorahosted/certmonger/.*Remote peer disconnected")
 
+        if self.machine.image == "fedora-34":
+            self.allow_journal_messages("json_object_get_string_member: assertion 'node != NULL' failed")
```

Comment:
Please add a "# HACK" comment here with that pointer, so that we know when to clean it up.